docs/src/Simulating.md

```diff
@@ -47,26 +63,32 @@ and the temperature is integrated for.
 When constructing a Domain object constant concentration species can be defined (list of species names).  
 During integration the derivatives with respect to time of these species will be kept at zero.  
 
-Sensitivity analysis can also be requested on the Domain object.  
+Forward sensitivity analysis can also be requested on the Domain object by setting `sensitivity=true`. Note that adjoint sensitivity analysis is usually much faster and can be done as a postprocessing analysis after the simulation is complete without a need to set `sensitivity=true` during the simulation (this is discussed in the Analysis section). 
```

Comment:
I don't think this is how we currently do this? I think we use `forwardsensitivities=true` in Reactor

Reply:
The `sensitivity=true` option in Domain doesn't do anything to my understanding? And we may want to remove it